### ISSUE TITLE
Make it so that when you add a new card the title input is focused.

### ISSUE THREAD

**Issue by jacob-and-nathan** [jacob-and-nathan/SoftwarePlanner#22]
Right now, this already happens on the first card you create, but NOT subsequent ones.

**Comment by carlosrrod** [comment]
I believe there is a method called `focus()` that does this. 
[I found this from MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/focus). You can try adding this to the `projectCardTitleInput`.

**Comment by jacob-and-nathan** [comment]
I think the issue is that even with `focus()` (I have tried it just now), it still seems to only work the first time you create a card...

**Comment by carlosrrod** [comment]
Yes you're right, I also tried experimenting with `focus()` and `autofocus` but it only seems to apply to the first card. It's difficult to target the input of each card, since they all created from the same function and share the same classes, id's, etc. 

What are your thoughts?

**Comment by jacob-and-nathan** [comment]
I have just noticed that the input has an ID applied to it instead of a CLASS. (an ID is for 1 element, but a CLASS works with any amount of elements.)
This might be the cause of the problem, so I will try and change it to a class and see what happens.
